.github/ISSUE_TEMPLATE/release_new_version.md

```diff
@@ -0,0 +1,70 @@
+---
+name: 'For maintainers: Release a new version'
+about: This is the Developer-only template which we use for tracking the process of future releases.
+title: 'Prepare Release 3.y.z'
+labels: release
+assignees: ''
+---
+
+**Target timeline**
+Schuled feature freeze / Start of translation process: <!-- Add a date, usually about 6-8 weeks from the last release, e.g. 2021-04-20 -->
+Approximate release date: <!-- Usually 2-3 weeks after starting translations, e.g. 2021-05-04 -->
+
+**Checklist**
+- [ ] Assign this issue to the release shepherd who is in charge of managing this checklist
+- [ ] Create a milestone and add all relevant issues
+- [ ] Ensure that all issues/PR targeted for this release are done by checking the milestone
+- [ ] Declare a code freeze. PRs can still be worked on and may get reviewed, but must not be merged unless agreed explicitly.
+- [ ] Write the Changelog
+- [ ] [Tag a beta release](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#steps-for-a-specific-release)
+- [ ] Announce the beta release on Github Discussions
+- [ ] Get feedback on the stability of the beta release
+- [ ] Website translations
+  - [ ] Create new branch `translate3_y_z` based on branch `release`
+  - [ ] Squash-merge `changes` into `translate3_y_z`
```

Comment:
I think there's a missing step we talked about: Open Issues for each language (so that we can track the status of translation, and so that translators can discuss issues if they want before they submit PRs). Assigning these to their respective translators also implies we need a list of who is volunteering to which language. Right now we have no Germans, for example. Translators then create PRs for both app and website if needed, linking them to their Issue.

Reply:
I've added the information about individual Issues, which was missing.
I'm leaving this comment unresolved for now as we still need a way to find out who is responsible for each language. `git log $LANG*.md` may help if there's nothing else?

Reply:
This is now done via the new `tools/create-translation-issues.sh`. It contains a static list of translators though. Will need to update that based on `git log` if required (this is a checklist entry on its own).